### PR TITLE
Improve "What clients are available?" section

### PR DESCRIPTION
**Pull request by r00ster91**
Preview: https://pr1334--matrix-org-previews.netlify.app

### CODE REVIEW

File: gatsby/src/pages/faq.js [hunk 1]
```diff
@@ -815,7 +815,15 @@ const Faq = ({ data }) => {
...
-                See also:{" "}
+                See also the following lists:{" "}
+                <a href="/clients">
+                  Clients
+                </a>
+                ,
+                <a href="/clients-matrix">
+                  Clients Matrix
+                </a>
+                , and
```
Comment (thibaultamartin) [inline gatsby/src/pages/faq.js:826]: When the website is built, the spaces are missing after the commas

File: gatsby/src/pages/faq.js [hunk 2]
```diff
@@ -826,8 +834,9 @@ const Faq = ({ data }) => {
                 macro.
               </p>
               <p>
-                The most popular and established client is Element, available on
-                web, desktop, Android and iOS.
+                The most popular and established client is
+                <a href="https://element.io">Element</a>
+                , available on web, desktop, Android and iOS.
```
Comment (thibaultamartin) [inline gatsby/src/pages/faq.js:839]: When the website is built, there is a space missing between "is" and Element

**Comment by r00ster91** [comment]
@thibaultamartin fixed, good catch!

File: gatsby/src/pages/faq.js [hunk 3]
```diff
@@ -815,7 +815,15 @@ const Faq = ({ data }) => {
...
-                See also:{" "}
+                See also the following lists:{" "}
+                <a href="/clients">
+                  Clients
+                </a>
+                ,{" "}
+                <a href="/clients-matrix">
+                  Clients Matrix
+                </a>
+                , and
```
Comment (thibaultamartin) [inline gatsby/src/pages/faq.js:826]: Missing a space here too, and we should be good :)

Comment (r00ster91) [reply gatsby/src/pages/faq.js]: Ugh, you're right! They're not that easy to spot.

**Comment by thibaultamartin** [comment]
Thanks for helping us tidying up the website :)

**Pull request merged**